### PR TITLE
Foundation/home

### PR DESCRIPTION
**Pull request by JonahMania**

### CODE REVIEW

**Comment by NicholasAntonov** [comment]
This should stay on one row, and `overflow-x: scroll;`

File: static/scss/home.scss [hunk 1]
```diff
@@ -0,0 +1,155 @@
...
+    }
+}
+
+#js-main-container {
+    display: flex;
+    flex-direction: column;
+
+    .book-detail {
+        .book-cover {
+            a {
```
Comment (NicholasAntonov) [inline static/scss/home.scss]: I think we can safely remove these empty rules, right?

File: static/scss/home.scss [hunk 2]
```diff
@@ -0,0 +1,155 @@
...
+}
+
+.book-list > div{
+    display: flex;
+    flex-wrap: wrap;
+    box-sizing: content-box;
+}
+
+.thewholebook {
+   border: none !important;
```
Comment (NicholasAntonov) [inline static/scss/home.scss]: Can we avoid using `!important`? Maybe find out what other class is adding that border and remove it?

File: static/scss/landingpage4.scss [hunk 3]
```diff
@@ -359,3 +359,8 @@ ul#as_seen_on {
         }
     }
 }
+
+#footer {
```
Comment (NicholasAntonov) [inline static/scss/landingpage4.scss]: We don't have an element with an id of `footer` anymore to my knowledge and a quick grep. Is this old code from moving something over?

**Comment by eshellman** [comment]
I'm still seeing 404 requests for the panel javascript

[08/May/2018 15:07:05] "GET /static/scss/$%7Bimage-base%7Dbg.png HTTP/1.1" 404 1694
[08/May/2018 15:07:05] "GET /static/images/favicon.ico HTTP/1.1" 200 2212
[08/May/2018 15:07:41] "GET /campaigns/unglued HTTP/1.1" 200 50325
[08/May/2018 15:07:41] "GET /static/scss/book_panel2.css HTTP/1.1" 304 0
[08/May/2018 15:07:41] "GET /static/scss/book_list.css HTTP/1.1" 200 6557
[08/May/2018 15:07:41] "GET /static/scss/lists.css HTTP/1.1" 200 417
[08/May/2018 15:07:41] "GET /static/scss/supporter_layout.css HTTP/1.1" 200 6758
[08/May/2018 15:07:41] "GET /static/js/toggle.js HTTP/1.1" 200 827
[08/May/2018 15:07:41] "GET /static/js/greenpanel.js HTTP/1.1" 404 1666
[08/May/2018 15:07:41] "GET /static/js/greenpanel.js HTTP/1.1" 404 1666

**Comment by eshellman** [comment]
is the "matrix" moving somewhere?

**Comment by eshellman** [comment]
"Donate" and "News" should be on top of "Latest Ungluing", don't you think?

**Comment by JonahMania** [comment]
We are planning to move the "matrix" to the faq page

**Comment by eshellman** [comment]
on the desktop version there's a "gutter" between the lanes of books and the activity column - maybe because the featured book space is too wide. should be fixed.

**Comment by eshellman** [comment]
also, can we have some separation between blocks like "donate" and "latest ungluing"

**Comment by JonahMania** [comment]
@eshellman Im not sure what you mean by a "gutter" could you post a screenshot?

**Comment by eshellman** [comment]
image

**Comment by eshellman** [comment]
also, I'm still getting:
[15/May/2018 12:04:00] "GET /static/scss/$%7Bimage-base%7Dbg.png HTTP/1.1" 404 1694

**Comment by eshellman** [comment]
I see the problem with the space between margin sections. What I meant was that there should be a *visible* space, i.e a gap to tell the user "these are 2 different things", the way there is on the current home page. this design has one monolithic blue side bar.

**Comment by eshellman** [comment]
image
vs,
image

**Comment by eshellman** [comment]
better!

**Pull request merged**